README.md

```diff
@@ -1409,6 +1456,110 @@ yarn rebuild
 
 This combines the top-level `prepare` and `build` scripts.
 
+You'll find the output in `packages/dev-frontend/build`.
+
+### Configuring your custom Dev UI
```

Comment:
Perhaps we should replace "Dev UI" by "Frontend" throughout the Readme.

Reply:
Good idea - done: https://github.com/liquity/dev/pull/529/commits/a88ac4964ed4a03beb43f4d7afdf007de0eb16b1